src/modules/commands/CommandHandler.ts

```diff
@@ -236,6 +237,30 @@ export class CommandHandler<T = unknown> {
       }
     }
 
+    if (message.text.startsWith(`${prefix}`)) {
+      const commandsAliases: string[] = []
+      this._commands.forEach((command) => {
+        commandsAliases.push(command.keyword, ...command.alias)
+      })
+
+      const fz = fuzzySet(commandsAliases)
+      const match = fz.get(message.text.slice(prefix.length))
+      if (match === null || match === undefined) {
```

Comment:
This can be simplified to `if (!match)`.

Reply:
Noice